blocks/lists/lists.js

```diff
@@ -0,0 +1,4 @@
+export default async function decorate(block) {
```

Comment:
This isn't needed - the default for all but the first image on the page is lazy, and we want the FCP to not be lazy.